connector/connect/server/src/main/scala/org/apache/spark/sql/connect/artifact/SparkConnectArtifactManager.scala

```diff
@@ -183,6 +183,9 @@ class SparkConnectArtifactManager(sessionHolder: SessionHolder) extends Logging
 
     // Clean up artifacts folder
     FileUtils.deleteDirectory(artifactPath.toFile)
+
+    // Clean up running queries
+    SparkConnectService.streamingSessionManager.cleanupRunningQueries(sessionHolder)
```

Comment:
I actually think it has to be placed under `expireSession`. This class is really all about the artifacts you add, and strning query cleanup is irrelevant.

Reply:
Thanks @HyukjinKwon, updated.